rll-examples/security-updates.sh

```diff
@@ -6,18 +6,53 @@
 # ...
 #
 
+# Run sudo apt-get|yum with retries if errors occur.
+# $@: full APT-GET(8) or yum(8) command
+#
+function package_handler() {
```

Comment:
As-is nothing in this function seems to specific to package managers in general, its a generic retry function -- leave it up to you if you want to rename it to "retry" or something. Obviously don't rename it if you see it handling specific error codes in the future.